.changeset/seven-balloons-attack.md

```diff
@@ -0,0 +1,41 @@
+---
+'xstate': major
+---
+
+Actor types can now be specified in the `.types` property of `createMachine`:
+
+```ts
+const machine = createMachine({
+  types: {} as {
+    actors: {
+      src: 'fetchData'; // src name (inline behaviors ideally inferred)
+      id: 'fetch1' | 'fetch2'; // possible ids
+      input: { foo: string };
+      output: { result: string };
```

Comment:
Before adding `input` and `output` here we should first add `TInput` and `TOutput` to the actor types.

Reply:
Done